### PR TITLE
Jupiter 10.12.0 plat 2908

### PR DESCRIPTION
**Pull request by shpiNi**

### CODE REVIEW

**Comment by MosheMaorKaltura** [comment]
Approved.

File: alpha/apps/kaltura/modules/extwidget/actions/playManifestAction.class.php [hunk 1]
```diff
@@ -447,7 +437,11 @@ protected function initSmilManifest($flavorAssets)
 
 	private function removeNotAllowedFlavors($flavorAssets)
 	{
-		$returnedFlavors = array();		
+		if(!$this->secureEntryHelper)
+			return $flavorAssets;
```
Comment (yuly-roberman) [inline alpha/apps/kaltura/modules/extwidget/actions/playManifestAction.class.php]: @shpiNi why do we allow a case where secureEntryHelper is not set?

Comment (shpiNi) [inline alpha/apps/kaltura/modules/extwidget/actions/playManifestAction.class.php]: @yuly-roberman we bypass it when a 'kt' token is passed.

Comment (yuly-roberman) [inline alpha/apps/kaltura/modules/extwidget/actions/playManifestAction.class.php]: @shpiNi so make sure to validate it also in initFlavorIds() function

**Pull request merged**